py_ecc/bls/hash.py

```diff
@@ -30,8 +36,31 @@ def hkdf_expand(prk: Union[bytes, bytearray], info: Union[bytes, bytearray], len
         text = previous + info + bytes([i + 1])
 
         # T(i + 1) = HMAC(T(i) || info || i)
-        previous = bytearray(hmac.new(prk, text, hashlib.sha256).digest())
+        previous = bytearray(hmac.new(prk, text, _sha256).digest())
         okm.extend(previous)
 
     # Return first `length` bytes.
     return okm[:length]
+
+
+def sha256(x: bytes) -> bytes:
+    m = _sha256()
+    m.update(x)
+    return m.digest()
```

Comment:
It seems we don't need to concatenate the `x` here, right?
Can we just use `return sha256(x).digest()`?